src/RTCDTMFSender.ts

```diff
@@ -0,0 +1,36 @@
+import { NativeModules, Platform } from "react-native";
+
+const { WebRTCModule } = NativeModules;
+const DEFAULT_DURATION = 300;
+const DEFAULT_INTER_TONE_GAP = 70;
+
+export default class RTCDTMFSender {
+    _peerConnectionId: number;
+    _senderId: string;
+    constructor(peerConnectionId: number, senderId: string) {
+        this._peerConnectionId = peerConnectionId;
+        this._senderId = senderId;
+    }
+
+    get canInsertDTMF(): boolean {
+        // Blocking!
+        return WebRTCModule.peerConnectionCanInsertDTMF(
```

Comment:
This is not a PC function but a sender one, can you please rename the native functions to senderCanInsertDTMF and senderInsertDTMF